### ISSUE TITLE
Error deploying 

### ISSUE THREAD

**Issue by peterglock** [blockapps/blockapps-ba#85]
I'm deploying the sample app on the same aws instance that stratos is running on. I get the following error:

```
ubuntu@ip-172-31-36-236:~/blockapps-ba$ SERVER=ip-172-31-36-236 npm run deploy

> blockapps-ba@2.1.0 deploy /home/ubuntu/blockapps-ba
> mocha server/dapp/dapp.deploy.js --config server/config/${SERVER:-localhost}.config.yaml -b && cp server/config/${SERVER:-localhost}.config.yaml ./config.yaml

/usr/bin/env: ‘node’: No such file or directory

npm ERR! Linux 4.4.0-1013-aws
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "run" "deploy"
npm ERR! node v4.2.6
npm ERR! npm  v3.5.2
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! blockapps-ba@2.1.0 deploy: `mocha server/dapp/dapp.deploy.js --config server/config/${SERVER:-localhost}.config.yaml -b && cp server/config/${SERVER:-localhost}.config.yaml ./config.yaml`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the blockapps-ba@2.1.0 deploy script 'mocha server/dapp/dapp.deploy.js --config server/config/${SERVER:-localhost}.config.yaml -b && cp server/config/${SERVER:-localhost}.config.yaml ./config.yaml'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the blockapps-ba package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     mocha server/dapp/dapp.deploy.js --config server/config/${SERVER:-localhost}.config.yaml -b && cp server/config/${SERVER:-localhost}.config.yaml ./config.yaml
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs blockapps-ba
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls blockapps-ba
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/ubuntu/blockapps-ba/npm-debug.log
```

How do I debug?

**Comment by peterglock** [comment]
Sorry, that was simple to fix https://github.com/nodejs/node-v0.x-archive/issues/3911

I've now got 'node' running but get this error:
```
ubuntu@ip-172-31-36-236:~/blockapps-ba$ SERVER=ip-172-31-36-236 npm run deploy

> blockapps-ba@2.1.0 deploy /home/ubuntu/blockapps-ba
> mocha server/dapp/dapp.deploy.js --config server/config/${SERVER:-localhost}.config.yaml -b && cp server/config/${SERVER:-localhost}.config.yaml ./config.yaml

/home/ubuntu/blockapps-ba/node_modules/blockapps-rest/lib/rest_5.js:40
class HttpError extends Error {
^^^^^

SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:374:25)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/ubuntu/blockapps-ba/node_modules/blockapps-rest/index.js:1:76)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/ubuntu/blockapps-ba/server/dapp/dapp.deploy.js:2:12)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at /home/ubuntu/blockapps-ba/node_modules/mocha/lib/mocha.js:231:27
    at Array.forEach (native)
    at Mocha.loadFiles (/home/ubuntu/blockapps-ba/node_modules/mocha/lib/mocha.js:228:14)
    at Mocha.run (/home/ubuntu/blockapps-ba/node_modules/mocha/lib/mocha.js:514:10)
    at Object.<anonymous> (/home/ubuntu/blockapps-ba/node_modules/mocha/bin/_mocha:480:18)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Function.Module.runMain (module.js:442:10)
    at startup (node.js:136:18)
    at node.js:966:3
```

**Comment by peterglock** [comment]
Last comment (hopefully). The version of nodejs included in the stratos aws image is v4.2.6.

I've upgraded to v7.10.1 and the app now deploys. I'll start to play now...
